app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -2,7 +2,7 @@
 
 class ManageIQ::Providers::Kubernetes::ContainerManager::Scanning::Job < Job
   PROVIDER_CLASS = ManageIQ::Providers::Kubernetes::ContainerManager
-  INSPECTOR_IMAGE = 'docker.io/fsimonce/image-inspector:v0.1.3'
+  INSPECTOR_IMAGE = 'docker.io/fsimonce/image-inspector:v1.0.z'
```

Comment:
@moolitayer it should be `openshift/image-inspector:v1.0.z`

Please also check that it's working.